packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/BasicInformationStep/BasicInformationForm/BasicInformationForm.tsx

```diff
@@ -95,13 +95,26 @@ const BasicInformationForm = ({
     }
 
     return chainTypes
-      .filter(
-        (chainType) =>
-          chainType.chainBase ===
-          (selectedCommunity.type === CommunityType.Cosmos
+      .filter((chainType) => {
+        const type =
+          selectedCommunity.type === CommunityType.Cosmos
             ? CommunityType.Cosmos
-            : CommunityType.Ethereum),
-      )
+            : CommunityType.Ethereum;
+
+        if (chainType.chainBase === type) {
+          return !(
+            window.location.hostname.includes('commonwealth.im') &&
+            [
+              'evmosdevci',
+              'csdkv1',
+              'csdkbeta',
+              'csdkv1ci',
+              'csdkbetaci',
+              'evmosdev',
+            ].includes(String(chainType.value))
+          );
+        }
+      })
```

Comment:
imo we should filter them out in `constants.ts` file right away,  where `chainTypes` is defined. You can add similar function like we have there   `.filter(particularChainNodes)` that will better describe why we filter them out (either descriptive variables or short comment)